src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,83 @@
+.clock {
+  width: 350px;
+  height: 350px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: url("../images/clock.png") center / cover;
+  border: 4px solid $arrow-color;
+  border-radius: 50%;
+  box-shadow: 0 -15px 15px $shadow-color,
+    inset 0 15px 15px $shadow-color;
+  position: relative;
+
+  &::before {
+    content: "";
+    position: absolute;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+    background: $arrow-color;
+    z-index: 3;
+  }
+
+  &__minute {
+    position: absolute;
+
+    &-minutes {
+      width: 190px;
+      height: 190px;
+      display: flex;
+      justify-content: center;
+      position: relative;
+      border-radius: 50%;
+
+      animation: time 3600s linear infinite;
+
+      &::before {
+        content: "";
+        position: absolute;
+        width: 4px;
+        height: 90px;
+        background: $arrow-color;
+        z-index: 2;
+        border-radius: $arrow-border-radius;
+      }
+    }
+  }
+
+  &__second {
+    position: absolute;
+
+    &-seconds {
+      width: 230px;
+      height: 230px;
+      display: flex;
+      justify-content: center;
+      position: relative;
+      border-radius: 50%;
+
+      animation: time 60s linear infinite;
+
+      &::before {
+        content: "";
+        position: absolute;
+        width: 2px;
+        height: 150px;
+        background: #eb4e4e;
+        border-radius: $arrow-border-radius;
+        z-index: 1;
+      }
+    }
+  }
```

Comment:
you have a lot of repeatable code with &__minute. So either make some placeholders/mixins or use a selector with common styles and modifiers to add difference